Bottom: b58f11894f05349410687ef47d2f178930d467e8
Top:    8d39935fdbb022bdbae64949a54d8eeac87c5b10
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 01:49:29 +0000

Refresh of more-caching-for-step-output

---

diff --git a/src/hid/step/step_writer.c b/src/hid/step/step_writer.c
index ab82b0d..7c9605d 100644
--- a/src/hid/step/step_writer.c
+++ b/src/hid/step/step_writer.c
@@ -113,13 +113,13 @@ make_3d_metric_step_geometric_representation_context (step_file *file)
 }
 
 static step_id
-lookup_or_create (GHashTable *hash, char *entity, char *content)
+lookup_or_create (step_file *file, GHashTable *hash, const char *entity, char *content)
 {
   step_id id;
 
   if ((id = GPOINTER_TO_INT (g_hash_table_lookup (hash, content))) != 0)
     {
-      g_free (line);
+      g_free (content);
       return id;
     }
